source/adminguide/service_offerings.rst

```diff
@@ -187,8 +187,17 @@ To create a new compute offering:
       system VM is running. Shared allocates from storage accessible via
       NFS.
 
-   -  **Provisioning type**: The type of disk that should be allocated. 
-      Local
+   -  **Provisioning type**: The type of disk that should be allocated.
+      Valid values are thin, sparse, fat. When using the VMWare hypervisor,
+      these values map as follows:
+
+      -  **Thin** - **Thin**.
+      -  **Sparse** - **Thick provisioned lazily zeroed**.
+      -  **Fat** -  **Thick provisioned eagerly zeroed**.
+      
+      The disk provisioning type strictness on VMWare iscontrolled with the zone level setting - **disk.provisioning.type.strictness**
+
```

Comment:
white space not required, as this stmt is related to the setting.

Reply:
Done